src/types/Todo.ts

```diff
@@ -0,0 +1,9 @@
+import { User } from "./User";
+
+export type Todo = {
+  id: number,
+  title: string,
+  completed: boolean,
+  userId: number,
+  user: User | null,
+};
\ No newline at end of file
```

Comment:
Missing newline at the end of file. It's a general convention to end a file with a newline because some Unix utilities don't behave as expected when this newline is missing.